electron-builder.config.js

```diff
@@ -48,7 +48,6 @@ const platformSpecificConfig = {
     linux: {
         ...baseConfig,
         linux: {
-            icon: "assets/Build/app-icon-dark.png",
```

Comment:
Would it work if we add the path to the new PNG file?

Reply:
Pretty sure I tried that and it didn't work I'll give that another go after work